### ISSUE TITLE
foolbox.adversarial

### ISSUE THREAD

**Issue by alimirzaeian** [bethgelab/foolbox#361]
I was using foolbox.adversarial module and I was wondering if it's possible to modify the calculated gradient by whatever attacks and then let the attacks resume with the modified gradients. Right now I could manage to switch between different parameters of the return object of foolbox.adversarial, like distance, However, I couldn't set the new gradients and resume the underlying attack. Can you guide me if there is any way to do so?

**Comment by jonasrauber** [comment]
If you want to overwrite / modify the gradient in some way, I recommend subclassing the model class you use (e.g. `TensorFlowModel` or `PyTorchModel`) and overwriting the gradient methods.
